### DISCUSSION TITLE
Include jbake headers

### DISCUSSION THREAD

**Discussion by grimsell** [docToolchain/docToolchain#952] [Q&A]
I've just started using doctoolchain to generate a documentation site using jbake.
I need some jbake headers in all asciidoc files. At least the jbake-menu header if I understand it correctly.

Is there a way to include the headers from a shared file? If I include it with a asciidoc include it won't work.

**Comment by grimsell** [comment]
Realised that jbake-menu will deafult to the top level folder of the page so I don't need to set any jbake headers in all pages.

**Comment by rdmueller** [comment]
Yes, exactly. That is the convenrion over configuration approach.
By now, there is even a mechanism to modify the approach.
